changelog/_6473.md

```diff
@@ -0,0 +1,5 @@
+## Updated
```

Comment:
we follow https://keepachangelog.com/en/1.1.0/ which doesn't know `Updated`, in this case it would be `Changed` I'd say